deployment/kubernetes/example/default_example_provisioner_generated.yaml

```diff
@@ -36,7 +36,7 @@ spec:
     spec:
       serviceAccountName: local-storage-admin
       containers:
-        - image: "quay.io/external_storage/local-volume-provisioner:v2.1.0"
+        - image: "quay.io/external_storage/local-volume-provisioner:v2.4.0"
```

Comment:
we've moved our image building to k8s.gcr.io now: k8s.gcr.io/sig-storage/local-volume-provisioner:v2.4.0

Reply:
Hi @msau42  I have addressed the changes .PTAL